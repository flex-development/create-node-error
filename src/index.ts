/**
 * @file Package Entry Point
 * @module create-node-error
 */

export {}
