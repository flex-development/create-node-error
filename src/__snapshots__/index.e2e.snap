// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:errnode > should expose public api 1`] = `
[
  "E",
  "codes",
  "syscodes",
  "ERR_ENCODING_NOT_SUPPORTED",
  "ERR_IMPORT_ASSERTION_TYPE_FAILED",
  "ERR_IMPORT_ASSERTION_TYPE_MISSING",
  "ERR_IMPORT_ASSERTION_TYPE_UNSUPPORTED",
  "ERR_IMPORT_ATTRIBUTE_MISSING",
  "ERR_IMPORT_ATTRIBUTE_TYPE_INCOMPATIBLE",
  "ERR_IMPORT_ATTRIBUTE_UNSUPPORTED",
  "ERR_INCOMPATIBLE_OPTION_PAIR",
  "ERR_INVALID_ARG_TYPE",
  "ERR_INVALID_ARG_VALUE",
  "ERR_INVALID_MODULE_SPECIFIER",
  "ERR_INVALID_PACKAGE_CONFIG",
  "ERR_INVALID_PACKAGE_TARGET",
  "ERR_INVALID_RETURN_VALUE",
  "ERR_INVALID_URL",
  "ERR_INVALID_URL_SCHEME",
  "ERR_METHOD_NOT_IMPLEMENTED",
  "ERR_MISSING_OPTION",
  "ERR_MODULE_NOT_FOUND",
  "ERR_NETWORK_IMPORT_DISALLOWED",
  "ERR_OPERATION_FAILED",
  "ERR_PACKAGE_IMPORT_NOT_DEFINED",
  "ERR_PACKAGE_PATH_NOT_EXPORTED",
  "ERR_UNHANDLED_ERROR",
  "ERR_UNKNOWN_BUILTIN_MODULE",
  "ERR_UNKNOWN_ENCODING",
  "ERR_UNKNOWN_FILE_EXTENSION",
  "ERR_UNKNOWN_MODULE_FORMAT",
  "ERR_UNSUPPORTED_DIR_IMPORT",
  "ERR_UNSUPPORTED_ESM_URL_SCHEME",
  "ERR_UNSUPPORTED_RESOLVE_REQUEST",
  "errors",
  "determineSpecificType",
  "formatList",
  "hideStackFrames",
  "isNodeError",
]
`;
